safe.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>SAFENET</title>
+  </head>
+  <body>
+    <!-- top nav bar -->
+    <nav class="top-nav">
+      <ul class="top-ul">
+        <li class="top-li">
+          <a href="#"><i class="fa-brands fa-facebook"></i></a>
+        </li>
+        <li class="top-li">
+          <a href=""><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li class="top-li">
+          <a href="#">English</a>
+        </li>
+        <li class="top-li">
+          <a href="">mypage</a>
+        </li>
+        <li class="top-li">
+          <a href="">logout</a>
+        </li>
+      </ul>
+    </nav>
+    <!-- main navbar  -->
+    <nav class="nav">
+      <div class="logdiv">
+        <img src="./images/logo.jpg" alt="logo" />
+        <a href="">safenet</a>
+      </div>
+      <ul class="nav-ul">
+        <li class="nav-li-active link"><a href="./about.html">About</a></li>
+        <li class="nav-li link"><a href="#">Program</a></li>
+        <li class="nav-li link"><a href="#">join</a></li>
+        <li class="nav-li link"><a href="#"> sponser</a></li>
+        <li class="nav-li link"><a href="#">News</a></li>
```

Comment:
- [ ] please add links to the different sections here so it makes it easier for a user to navigate